pkg/status/workstatus.go

```diff
@@ -90,6 +90,11 @@ func updateObjectStatus(ctx context.Context, objectIdentifier util.ObjectIdentif
 	listers util.ConcurrentMap[schema.GroupVersionResource, cache.GenericLister], wdsDynClient dynamic.Interface) error {
 	logger := klog.FromContext(ctx)
 
+	if util.WEC2WDSExceptions.Has(objectIdentifier.GVK) {
+		logger.V(4).Info("Status from WEC shouldn't have authority to overwrite status in WDS", "object", objectIdentifier)
```

Comment:
This is a user error and should be reported to the user. OK if that is done in a later PR.